packages/suite/src/views/wallet/trading/common/TradingForm/TradingNetworkReserveBanner.tsx

```diff
@@ -0,0 +1,57 @@
+import { CryptoId } from 'invity-api';
+
+import { cryptoIdToNetwork, getNetworkReserve } from '@suite-common/trading';
+import { selectIsNetworkReserveEnabled } from '@suite-common/wallet-core';
+import { Banner } from '@trezor/components';
+
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite/Translation';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+
+interface TradingNetworkReserveBannerProps {
+    crypto?: CryptoId;
```

Comment:
can it accept `NetworkSymbol` and `tokenAddress`? If `tokenAddress` is defined, you will not show it